lumibot/data_sources/pandas_data.py

```diff
@@ -203,7 +210,7 @@ def get_last_price(self, asset, quote=None, exchange=None):
     def get_last_prices(self, assets, quote=None, exchange=None, **kwargs):
         result = {}
         for asset in assets:
-            result[asset] = self.get_last_price(asset, quote=quote, exchange=exchange)
+            result[asset] = self.get_last_price(asset, timestep=self._timestep, quote=quote, exchange=exchange)
```

Comment:
Like korbit says, DataSource.get_last_price doesn't take timestep.  That might be a worthwhile change but rather than trying to include that in this PR I'd suggest making that a separate issue.